src/foam/core/AbstractArrayPropertyInfo.java

```diff
@@ -19,6 +19,15 @@
 public abstract class AbstractArrayPropertyInfo
   extends AbstractPropertyInfo {
 
+  public int compareValues(Object[] b1, Object[] b2) {
+    if (b1 == b2) return 0;
```

Comment:
Spacing should be:
  if ( b1 == b2 ) return 0;

Reply:
Fixed